sdk/synapse/Microsoft.Azure.Synapse/Directory.Build.Data.props

```diff
@@ -0,0 +1,133 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Setup default project properties -->
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <!--
+      Disable NU5105 NuGet Pack warning that the version is SemVer 2.0.
+      SemVer 2.0 is supported by NuGet since 3.0.0 (July 2015) in some capacity, and fully since 3.5.0 (October 2016).
+    -->
+    <NoWarn>$(NoWarn);NU5105</NoWarn>
+    <!--
+      Disable some FxCop rules
+     -->
+    <NoWarn>
+      $(NoWarn);
+      CA1716; <!-- avoid using method names that are reserved in other languages -->
+      CA1308; <!-- prefer ToUpperInvariant  -->
+      CA1819; <!-- Properties should not return arrays -->
+      CA1710; <!-- Use Collection sufix -->
+      CA1028; <!-- Enum backing type should be int -->
+      CA1032; <!-- Add default exception ctors -->
+      CA1063; <!-- Dispose implementation -->
+      CA1066; <!-- Implement IEquatable -->
+      CA1815; <!-- Equality members on structs -->
+      CA2007; <!-- Do not directly await a Task without calling ConfigureAwait (replaced by AZC0012) -->
+      CA2231; <!-- Override Equality operators -->
+      CA2237; <!-- [Serializeable] on exceptions -->
+      CA2225; <!-- Provide alternative to implicit operators -->
+      CA1714; <!-- Flags should have plural names -->
+    </NoWarn>
```

Comment:
Any reason to disable all of these?

Reply:
> Any reason to disable all of these?

Hi @AlexGhiondea , This file is not created by me. I copied the file from eng\Directory.Build.Data.props by following the guide https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#project-structure

Reply:
> Any reason to disable all of these?

Removed the file